docs/playground.html

```diff
@@ -1,70 +1,65 @@
+	<link data-name="vs/editor/editor.main" rel="stylesheet"
+		href="https://microsoft.github.io/monaco-editor/node_modules/monaco-editor/min/vs/editor/editor.main.css">
+	<link rel="stylesheet" href="/static/shared/iframeeditor.css">
+	<link rel="stylesheet" href="/static/playground/playground.css">
 </head>
 
-<body class="playground-page">
-    <pre data-preload="/playground/samples/basic/hello-world/sample.ts"></pre>
+<body>
+	<nav class="ui menu inverted fixed borderless">
+		<div class="menu left">
+			<div class="ui item">
+				MakeCode Blocks Playground
+			</div>
+		</div>
 
-    <nav class="ui menu inverted fixed borderless">
-        <div class="menu left">
-            <div class="ui item">
-                MakeCode Playground
-            </div>
-        </div>
+		<div class="menu right">
+			<a href="/defining-blocks" target="_blank" class="ui item">
+				Defining Blocks
+			</a>
+		</div>
+	</nav>
 
-        <div class="menu right">
-            <a href="/docs" class="ui item">
-                Home
-            </a>
-            <a href="/playground" class="ui item">
-                Playground
-            </a>
-            <a href="/defining-blocks" class="ui item">
-                Help defining blocks
-            </a>
-            <a href="https://github.com/microsoft/pxt-sample" class="ui item" style="background-color:black">
-                Sample target
-            </a>
-        </div>
-    </nav>
-    <div id="loading">
-        <div class="spinner">
-            <div class="rect1"></div>
-            <div class="rect2"></div>
-            <div class="rect3"></div>
-            <div class="rect4"></div>
-            <div class="rect5"></div>
-        </div>
-    </div>
+	<div class="content">
+		<div>
+			<div id="sample-container" class="ui action input">
+				<button class="ui small primary labelled icon left attached button" id="run-button">
+					<i class="play icon"></i>
+					Run
+				</button>
+				<select id="sample-select" class="ui dropdown"></select>
+			</div>
+			<div id="run-container" class="ui action input">
+				<select id="endpoint-select" class="ui dropdown"></select>
+			</div>
+		</div>
+		<div class="editor-row">
+			<div id="monaco-editor">
+				<div id="container" style="border:1px solid grey"></div>
+			</div>
+			<div id="makecode-editor" style="border:1px solid grey"></div>
+		</div>
+	</div>
 
-    <section id="playground">
-    </section>
 
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.min.js" integrity="sha256-wS9gmOZBqsqWxgIVgA8Y9WcQOa7PgSIX+rPA0VL2rbQ="
-        crossorigin="anonymous"></script>
-
-    <!-- @include playground-apptrackingweb.html -->
-    <!-- @include apptracking.html -->
+	<script type="text/javascript" src="https://pxt.azureedge.net/blob/795d8506c80a04f5ca26f577a8d6152e2fa3e7a6/doccdn/pxtweb.js"></script>
+    <!-- @include tracking.html -->
```

Comment:
The old page was using a different tracking template, why the change? Also do you know why it was including both apptracking and playground-apptracingweb?

Reply:
tracking.html is appinsights + pxtweb and apptracking is appinsights alone, and playground-apptrackingweb was pxtweb alone SO...... i don't know why it had a separate playground-apptracking file but this should add up to the same thing (appinsights + pxtweb). i think i can actually delete the pxtweb script tag here so i'll do that along with addressing the other comments!